docs/getting-started/installation.md

```diff
@@ -38,8 +38,17 @@ cargo install quickwit-cli
 
 ## Use the docker image
 
-Coming soon.
+If you use docker, this might be one of the quickest way to get going. 
+The following command will pull the image from [dockerhub](https://hub.docker.com/r/quickwitinc/quickwit)
+and gets you right in the running container ready to execute Quickwit commands.
 
+```bash
+docker run -it quickwitinc/quickwit
```

Comment:
This example needs to be improved so that users know how to mount a local directory into the image and can start indexing and querying indexes.

Reply:
Yes definitely if we are using quickwit as entrypoint